### PR TITLE
Removes "an entirely redundant second thing called the BEPIS".

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=25888
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=25889

As necessitated by the above two threads, the aforementioned machine is being removed in it's entirety. So as to not lose any of the content people created, all designs have been moved onto existing techs, or moved onto the standard techweb in order to meet orange's new design documentation he unveiled today (3/11/2020). Science has also had the machine itself removed from all major maps in order to free up space for whatever future form this new science rework takes.

Hopefully all the content design locations will be alright with everyone who contributed to this project. On the off chance you want something moved to a different tech or you feel it's placed too early/late in the techweb, please comment below so as to now crowd any of the policy threads. Let me know if anything else needs to be balanced, as I imagine all of economy and science should be considered in the removal of the largest money sink we have on station. Even i miss things on occasion but i believe everything is covered here.

## Changelog
:cl:
del: The BEPIS has been returned to it's resting place on Research Station 64, following it's reconstruction after a deer related nuclear incident.
balance: All BEPIS Techs and BEPIS minor rewards have been placed on the techweb.
/:cl:

### CODE REVIEW

**Comment by Couls** [comment]
mom, dad please stop fighting you're tearing me apart

**Comment by HomocidlePancakes** [comment]
The amount of trust you put in to the orangeman. Can’t tell whether it’s a good thing or not.

**Comment by XDTM** [comment]
The nanite replication protocols seem to be missing

Make sure they also get a nanite research point cost

File: code/modules/research/designs/mining_designs.dm [hunk 1]
```diff
@@ -131,3 +131,13 @@
...
+
+/datum/design/diamondpen
+	name = "Diamond Survival Pen"
+	desc = "A pen that has built in chemical reflectors that watchers crave. Throw it in the direction of one to draw it's fire!"
+	id = "dpen"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron=10, /datum/material/diamond=100, /datum/material/titanium = 10)
+	build_path = /obj/item/pen/survival
+	category = list("Mining Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_CARGO
```
Comment (Dennok) [inline code/modules/research/designs/mining_designs.dm]: Maybe better add it to service or all deps too? Its not propertly mining tool. This survival pen extrimely slow pick that can save life if you.... drop on lavaland?

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -1009,8 +1083,6 @@
 	design_ids = list("kickstart_nanites","factory_nanites","tinker_nanites","offline_nanites")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000, TECHWEB_POINT_TYPE_NANITES = 2500)
```
Comment (Dennok) [inline code/modules/research/techweb/all_nodes.dm]: Protocols here
and it have nanite prise

File: code/modules/research/designs/mining_designs.dm (hunk 1 above)
Comment (ATH1909) [reply code/modules/research/designs/mining_designs.dm]: honestly, I'd be fine with it being available at all protolathes

File: code/modules/research/techweb/all_nodes.dm (hunk 2 above)
Comment (XDTM) [reply code/modules/research/techweb/all_nodes.dm]: ah, my bad, i was doing a word search on replication protocols

**Comment by optimumtact** [comment]
wait until the poll

**Pull request closed without merging**